src/components/studies/participants/ParticipantManager.tsx

```diff
@@ -419,7 +419,10 @@ const ParticipantManager: FunctionComponent<ParticipantManagerProps> = () => {
     })
   const [isAllSelected, setIsAllSelected] = React.useState(false)
 
+  const [forceSearchReset, setForceSearchReset] = React.useState(false)
```

Comment:
Do you need an additional state variable? If you need to clear when the tab is changed can you just pass the current tab as a property to the search control and have your reset logic based on that?